apps/web/components/icons/DatabaseIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import type {SvgIconProps} from './SvgIcon'
+import {SvgIcon} from './SvgIcon'
+
+export function DatabaseIcon(props: SvgIconProps) {
+  return (
+    <SvgIcon {...props}>
+      <path
+        fillRule="evenodd"
```

Comment:
need to keep this for this glyph

Reply:
Nit: would put this as a comment in code

Reply:
Thanks, Rob! 4d8fedf